### DISCUSSION TITLE
TypeError for one url in CurlResponse

### DISCUSSION THREAD

**Discussion by Huluti** [symfony/symfony#59168] [Q&A] [answered]
### Symfony version(s) affected

7.2.0

### Description

This URL seems to cause issue one issue in CurlResponse:
`https://dictionary-cambridge-org.translate.goog/dictionary/english/state-of-mind?_x_tr_sl=en&_x_tr_tl=fr&_x_tr_hl=fr&_x_tr_pto=rq`

Errors:
`Warning: Undefined array key "connection"`

`TypeError(code: 0): array_map(): Argument #2 ($array) must be of type array, null given at vendor/symfony/http-client/Response/CurlResponse.php:319)"} []`

### How to reproduce

Don't know yet which part of my code involve CurlResponse

**Comment by xabbuh** [comment, marked as answer]
This is probably the same as #59039.

**Reply by Huluti** [reply]
Oh it seems yes thanks!
